cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -77,17 +77,23 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
     it('should have href that matches assetURI for 1st related content link', () => {
       cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
         const numRelatedContentGroups = body.relatedContent.groups.length;
-
-        if (numRelatedContentGroups > 0) {
+        const assetUriString = body.metadata.locators.assetUri;
+        if (
+          numRelatedContentGroups > 0 &&
+          assetUriString.split('/').length <= 5
```

Comment:
Let's create a function at the top level which determines if URL is legacy asset:

```
const isLegacyAsset = url => {
  return url.split('/').length > 4;
}
```

The refer to it in this condition:

```
if (numRelatedContentGroups > 0 && !isLegacyAsset(assetUri)) {
```

Reply:
Changed, different method. Didn't see comment before working on it